phpunit/bootstrap.php

```diff
@@ -48,7 +49,13 @@
 
 require_once dirname(__DIR__) . '/vendor/autoload.php';
 
-$kernel = new Kernel('testing');
+// Make sure cached content like twig template is cleared before running the tests.
+// It seems calling $cache_manager->resetAllCaches(); mess up with the kernel
+// leading to some issues. It is safer to use the console directly as it goes
+// throught another process.
+exec("bin/console cache:clear --env='testing");
```

Comment:
Not sure how everyone feel about that ?

I first tried to use `$cache_manager->resetAllCaches()` (done after the kernel is booted) but it mess up with things (some session warnings + the tester plugin setup being executed 3 times instead of once).

Going thought the console make it so the cache clearing action in done in a separate process which avoid these issues.
I guess the kernel does not like having its cache cleared when it is running (but maybe this is fixable).

Reply:
This only affects testing; so this should not be a problem since test files directory in not the same as the app one.